src/main/java/se/liu/ida/hefquin/engine/wrappers/graphqlwrapper/query/impl/GraphQLQueryImpl.java

```diff
@@ -33,7 +33,8 @@ public String toString() {
         return query;
```

Comment:
I cannot leave the comment in the respective line, but please, use a `StringBuilder` object instead of concatenating Strings. Using `+=` for strings is very inefficient.